packages/docs-reanimated/docs/guides/worklets.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: worklets
+sidebar_label: Worklets
+sidebar_position: 1
+---
+
+# Worklets
+
+Worklets are short-running JavaScript functions that can be run on the [UI thread](/docs/fundamentals/glossary#ui-thread). Reanimated uses worklets to update view properties and react to events on the UI thread.
+
+You can create your own worklets using the `"worklet";` directive at the top of a function.
```

Comment:
`"worklet";` or `'worklet';`?

Reply:
depends on your prettier config

Reply:
We use single quotes in our codebase so let's stick with it here as well.